packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.internal.tsx

```diff
@@ -1,36 +1,54 @@
 import React from 'react';
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
-import { CallToActionCard } from './CallToActionCard';
-import { select, text } from '@storybook/addon-knobs';
+import { CallToActionCard, CallToActionCardProps } from './CallToActionCard';
+import { NOOP_CONTROL } from '../../../.storybook/preview';
+import { Story } from '@storybook/react';
 import { Button } from '../Button/Button';
 import { action } from '@storybook/addon-actions';
 
 export default {
   title: 'Layout/CallToActionCard',
   component: CallToActionCard,
+  parameters: {
+    knobs: {
+      disabled: true,
+    },
+  },
+  argTypes: {
+    Element: { control: { type: 'select', options: ['button', 'custom'] } },
+    className: NOOP_CONTROL,
+    callToActionElement: NOOP_CONTROL,
+    theme: NOOP_CONTROL,
+  },
 };
 
-export const basic = () => {
+interface StoryProps extends Partial<CallToActionCardProps> {
+  Element: string;
+  H1Text: string;
+  buttonText: string;
+}
+
+export const basic: Story<StoryProps> = (args) => {
```

Comment:
```suggestion
export const Basic: Story<StoryProps> = (args) => {
```
The stories are basically React components, so should be capitalised. I know we have a lot of lowercased story names in the code, but we're trying to fix those.

Reply:
Alright. Thanks! @Clarity-89